pkgs/development/tools/go-jet/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "go-jet";
+  version = "2.9.0";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = "jet";
+    rev = "v${version}";
+    sha256 = "sha256-1kJaBLZIunexjxjOy55Nw0WMEhrSu+ptMbWVOJ1e5iA=";
+  };
+
+  vendorSha256 = "sha256-mhH9P3waINZhP+jNg3zKlssIL1ZO5xOBHp19Bzq/pSQ=";
+
+  subPackages = [ "cmd/jet" ];
+
+  tags = ["mysql" "golang" "postgres" "sql" "database" "code-generator" "sqlite" "postgresql" "mariadb" "sql-query" "codegenerator" "typesafe" "sql-builder" "datamapper" "code-completion" "sql-queries" "cockroachdb" "sql-query-builder" "sqlbuilder" "typesafety" ];
```

```suggestion
  tags = [
    "mysql"
    "golang"
    "postgres"
    "sql"
    "database"
    "code-generator"
    "sqlite"
    "postgresql"
    "mariadb"
    "sql-query"
    "codegenerator"
    "typesafe"
    "sql-builder"
    "datamapper"
    "code-completion"
    "sql-queries"
    "cockroachdb"
    "sql-query-builder"
    "sqlbuilder"
    typesafety"
  ];
```

we don't really have a line length but thats a bit long